VSharp.SILI.Core/Types.fs

```diff
@@ -77,7 +81,7 @@ and [<CustomEquality;CustomComparison>]
                 match other with
                 | :? typeId as other ->
                     match x, other with
-                    | Explicit h1, Explicit h2 -> compare h1.Inheritor.MetadataToken h2.Inheritor.MetadataToken
+                    | Explicit h1, Explicit h2 -> compare (hash h1.Inheritor) (hash h2.Inheritor) //TODO: when MONO will be better, use MetadataToken
```

Comment:
Change hash to MetadataToken when https://github.com/mono/mono/issues/10127 is fixed.